consensus/dummy/consensus.go

```diff
@@ -263,6 +263,9 @@ func (self *DummyEngine) verifyHeader(chain consensus.ChainHeaderReader, header
 		if err := eip4844.VerifyEIP4844Header(parent, header); err != nil {
 			return err
 		}
+		if *header.BlobGasUsed > 0 { // VerifyEIP4844Header ensures BlobGasUsed is non-nil
+			return fmt.Errorf("blobs not enabled on avalanche networks: have %d, expected 0", *header.BlobGasUsed)
```

Comment:
This error is a bit misleading because it implies a number of blobs. `: used %d blob gas, expected 0"` is more precise.